Procfile

```diff
@@ -0,0 +1 @@
+web: yarn serve-prod
```

Comment:
I don't know whether we'd need this or not, but I added back just to err on the side of having all potential build docs in master.